gulpfile.js

```diff
@@ -38,7 +36,7 @@ gulp.task ('css', function () {
 
 gulp.task ('zip', ['js', 'css'], function () {
   var version = require ('./package.json').version
-  
+
```

Comment:
Could you please revert these formatting changes? We want each commit’s diff to contain only those changes that the commit is related to. We usually ask to move such formatting changes to a separate commit, but as we now have #60 that will do them anyway, it’s OK to just revert them.